SDK/AppCenterData/Microsoft.AppCenter.Data.Android.Bindings/Additions/AboutAdditions.txt

```diff
@@ -0,0 +1,48 @@
+﻿Additions allow you to add arbitrary C# to the generated classes
```

Comment:
Can you delete these info txt files?

Reply:
These are in the other modules, I assumed we wanted them for consistency